### PR TITLE
Sysinfo extend

### PR DESCRIPTION
**Pull request by narasimhan-v**

### CODE REVIEW

**Comment by narasimhan-v** [comment]
```
# ./avocado-setup.py --bootstrap
07:51:27 INFO    : Check for environment
07:51:27 INFO    : Creating temporary mux dir
07:51:27 INFO    : Creating Avocado Config
07:51:27 INFO    : Bootstrapping Avocado
07:51:27 INFO    : Cloning the repo: avocado in /root/sim/tests/avocado
07:51:30 INFO    : Installing avocado from /root/sim/tests/avocado
07:51:36 INFO    : Cloning the repo: avocado-misc-tests in /root/sim/tests/tests/avocado-misc-tests
07:51:40 WARNING : Avocado varianter_yaml_to_mux plugin not installed
07:51:40 INFO    : Installing optional plugin: varianter_yaml_to_mux
07:51:41 INFO    : Removing temporary mux dir

```
```
# avocado run /bin/true
JOB ID     : cb4437c6692e3742989615958c3bdd308510847c
JOB LOG    : /root/sim/tests/results/job-2020-05-21T07.52-cb4437c/job.log
 (1/1) /bin/true: PASS (17.01 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 17.19 s
JOB HTML   : /root/sim/tests/results/job-2020-05-21T07.52-cb4437c/results.html
```
```
# grep sysinfo /root/sim/tests/results/job-2020-05-21T07.52-cb4437c/job.log
2020-05-21 07:52:23,955 job              L0519 INFO |  'diff.filter': ['cmdline', 'time', 'variants', 'results', 'config', 'sysinfo'],
2020-05-21 07:52:23,962 job              L0519 INFO |  'sysinfo.collect.enabled': 'True',
2020-05-21 07:52:23,962 job              L0519 INFO |  'sysinfo.collect.sysinfodir': None,
2020-05-21 07:52:24,057 sysinfo          L0409 INFO | Commands configured by file: /root/sim/tests/config/sysinfo/commands
2020-05-21 07:52:24,058 sysinfo          L0420 INFO | Files configured by file: /root/sim/tests/config/sysinfo/files
2020-05-21 07:52:24,058 sysinfo          L0440 INFO | Profilers configured by file: /root/sim/tests/config/sysinfo/profilers
2020-05-21 07:52:24,346 sysinfo          L0173 DEBUG| Not logging 'dmidecode' (command 'dmidecode' was not found)
2020-05-21 07:52:30,906 sysinfo          L0105 DEBUG| Not logging /proc/pci (file does not exist)
2020-05-21 07:52:30,934 sysinfo          L0173 DEBUG| Not logging 'brctl show' (command 'brctl' was not found)
2020-05-21 07:52:31,202 sysinfo          L0173 DEBUG| Not logging 'dmidecode' (command 'dmidecode' was not found)
2020-05-21 07:52:40,739 sysinfo          L0105 DEBUG| Not logging /proc/pci (file does not exist)
2020-05-21 07:52:40,763 sysinfo          L0173 DEBUG| Not logging 'brctl show' (command 'brctl' was not found)
```

File: config/sysinfo/commands [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+lspci -vvnn
+gcc --version
+ld --version
+hostname
+uptime
+dmidecode
+ifconfig -a
+brctl show
+ip link
+numactl --hardware show
```
Comment (sathnaga) [inline config/sysinfo/commands:13]: what is the current result if the dependency package is not installed in system for the commands?

Comment (narasimhan-v) [reply config/sysinfo/commands:13]: @sathnaga They are simply logged in the job.log like this:
```
2020-05-21 07:52:24,346 sysinfo          L0173 DEBUG| Not logging 'dmidecode' (command 'dmidecode' was not found)
2020-05-21 07:52:30,906 sysinfo          L0105 DEBUG| Not logging /proc/pci (file does not exist)
2020-05-21 07:52:30,934 sysinfo          L0173 DEBUG| Not logging 'brctl show' (command 'brctl' was not found)
2020-05-21 07:52:31,202 sysinfo          L0173 DEBUG| Not logging 'dmidecode' (command 'dmidecode' was not found)
2020-05-21 07:52:40,739 sysinfo          L0105 DEBUG| Not logging /proc/pci (file does not exist)
2020-05-21 07:52:40,763 sysinfo          L0173 DEBUG| Not logging 'brctl show' (command 'brctl' was not found)
```

Comment (sathnaga) [reply config/sysinfo/commands:13]: Not harmful...

Comment (narasimhan-v) [reply config/sysinfo/commands:13]: Definitely not harmful

**Comment by sathnaga** [comment]
@narasimhan-v I guess `sosreport` or equivalent for other distros, will have the super set of logs right, 
should we add that instead of hand picking individual commands?

**Comment by narasimhan-v** [comment]
> @narasimhan-v I guess `sosreport` or equivalent for other distros, will have the super set of logs right, 
> should we add that instead of hand picking individual commands?

@sathnaga sosreport / supportconfig are only for failing tests. These are for all tests.

**Comment by PraveenPenguin** [comment]
@sathnaga can this be in so we can leverage this for our recent runs

**Pull request merged**